pkg/kv/kvclient/kvtenant/connector.go

```diff
@@ -880,6 +880,24 @@ func (c *connector) HotRangesV2(
 	return resp, nil
 }
 
+// DownloadSpan implements the serverpb.TenantStatusServer interface
+func (c *connector) DownloadSpan(
+	ctx context.Context, req *serverpb.DownloadSpanRequest,
+) (*serverpb.DownloadSpanResponse, error) {
+	if !roachpb.IsSystemTenantID(c.tenantID.InternalValue) {
+		return nil, status.Errorf(codes.PermissionDenied, "only the system tenant can issue download span requests")
+	}
```

Comment:
sorry i dont think i follow, is the suggestion to do `if c.tenantID.InternalValue != roachpb.SystemTenantID` instead of using the helper `if !roachpb.IsSystemTenantID(c.tenantID.InternalValue)`?